frontend/src/components/page_content/configuration-content.js

```diff
@@ -1,9 +1,37 @@
 import React from "react";
+import Style from "style-it";
+
+import EmergencySavings from "../page_content/configuration/emergency-savings";
+import Protection from "../page_content/configuration/protection";
+import Debt from "../page_content/configuration/debt";
+import Retirement from "../page_content/configuration/retirement";
+import Budgeting from "../page_content/configuration/budgeting";
+import { resetStep } from "../../redux/actions";
+import { connect } from "react-redux";
 
 class ConfigurationContent extends React.Component {
+  onSubmit() {
+    // submit form
+  }
+
+  componentDidMount() {
```

Comment:
i dont think this is necessary because its reset already in the page template component